Directory.Build.props

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <PropertyGroup>
+    <LangVersion>7.2</LangVersion>
+    <DefaultItemExcludes>$(DefaultItemExcludes);*.DotSettings;*.ncrunchproject</DefaultItemExcludes>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Authors>Roland Pheasant</Authors>
+    <Owners>Roland Pheasant</Owners>
+    <PackageLicenseUrl>http://opensource.org/licenses/MIT</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/RolandPheasant/DynamicData</PackageProjectUrl>
+    <Copyright>Copyright Roland Pheasant 2011-2018</Copyright>
+    <PackageTags>DynamicData;Dynamic;Data;Rx;Reactive;Observable;Cache;Binding;ObservableCache;ObservableList;ObservableCollection;Collection;Linq</PackageTags>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
```

Comment:
Probably want <GenerateDocumentationFile>true</GenerateDocumentationFile> here

Reply:
On `GeneratePackageOnBuild` ? It was false previously and I left it that way. The `appveyor.yml` script packs the NuGet explicitly in a separate step.

This can be changed, of course. I'd suggest this if you want to simplify the build script:

```XML
<GeneratePackageOnBuild>False</GeneratePackageOnBuild>
<GeneratePackageOnBuild Condition="'$(Configuration)' == 'Release'">True</GeneratePackageOnBuild>
```

Reply:
Just helps with intellsense normally.

Reply:
Ooh, `GenerateDocumentationFile`. I didn't notice you've edited your comment.

If you move it to `Directory.Build.props`, you'll get a *ton* of warnings:

> warning CS1591: Missing XML comment for publicly visible type or member ...

So let's keep `GenerateDocumentationFile` in the two projects that need it. No need to document tests anyway 😉